code/modules/cm_marines/dropship_ammo.dm

```diff
@@ -321,6 +324,22 @@
 		cell_explosion(impact, 450, 100, EXPLOSION_FALLOFF_SHAPE_EXPONENTIAL, null, create_cause_data(initial(name), source_mob)) //Insane fall off combined with insane damage makes the Keeper useful for single targets, but very bad against multiple.
 		qdel(src)
 
+/obj/structure/ship_ammo/rocket/bunkerbuster
+	name = "\improper GBU-67-EP 'Paveway XXI'"
+	desc = "The GBU-67-EP 'Paveway XXI' takes the GBU-67 LGB and mates it with an enhanced penetrator warhead that allows it to strike targets dozens of meters underground."
+	icon_state = "bunkerbuster"
+	travelling_time = 20 //A fast payload due to its very tight blast zone
```

Comment:
is this in seconds? if so, use the SECONDS macro

Reply:
So every single other weapon in this .dm functions with this code, I'm just duplicating preexisting code.  Overhauling all of the CAS weapons seems beyond the scope of this PR.